### DISCUSSION TITLE
x6 画布添加标尺工具

### DISCUSSION THREAD

**Discussion by zlecheng** [antvis/X6#4431] [Show and tell]
![image](https://github.com/user-attachments/assets/99cf6cd4-9f00-4984-a54a-78eb72629885)

https://www.yuque.com/sxd_panda/antv/graph_rule

**Comment by zlecheng** [comment]
给画布添加标尺

**Reply by Yeti-xxx** [reply]
看起来挺不错的！如果通过scale事件调节标尺的刻度就更好了

**Comment by zlecheng** [comment]
这个自己实现吧

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年10月25日 17:49 |
| 收件人 | antvis/X6 ***@***.***> |
| 抄送人 | zlecheng ***@***.***>,
State ***@***.***> |
| 主题 | Re: [antvis/X6] x6 画布添加标尺工具 (Discussion #4431) |

看起来挺不错的！如果通过scale事件调节标尺的刻度就更好了
